keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -452,94 +462,200 @@ change are understandable. This may include API specs (though not always
 required) or even code snippets. If there's any ambiguity about HOW your
 proposal will be implemented, this is the place to discuss them.
 -->
+
 ### Overview
+
 Windows privileged containers will be implemented with [Job Objects](https://docs.microsoft.com/en-us/windows/win32/procthread/job-objects), a break from the previous container model using server silos. Job objects provide the ability to manage a group of processes as a group, and assign resource constraints to the processes in the job. Job objects have no process or file system isolation, enabling the privileged payload to view and edit the host file system with the correct permissions, among other host resources. The init process, and any processes it launches or that are explicitly launched by the user, are all assigned to the job object of that container. When the init process exits or is signaled to exit, all the processes in the job will be signaled to exit, the job handle will be closed and the storage will be unmounted.
 
+Because Windows privileged containers will work much differently than Linux privileged containers they will be referred to as **HostProcess** containers in kubernetes specs and user-facing documentation. Hopefully this will encourage users to seek documentation to better understand the capabilities and behaviors of these privileged containers.
+
 ![Privileged Container Diagram](Privileged.png)
 
 #### Networking
 - The container will be in the host’s network namespace (default network compartment) so it will have access to all the host’s network interfaces and have the host's IP as well.
+
 #### Resource Limits
 - Resource limits (disk, memory, cpu count) will be applied to the job and will be job wide. For example, with a limit of 10 MB is set for the job, if every process in the jobs memory allocations added up exceeds 10 MB this limit would be reached. This is the same behavior as other Windows container types. These limits would be specified the same way they are currently for whatever orchestrator/runtime is being used.
```

Comment:
For disk limits, it is counting the container image space, or the entire host system?

Reply:
Disk limits would track all file system operations performed by processes in the container regardless of what the access (either host resources or resources in the container volume). I can add a line here so it is captured in the KEP.

Reply:
Can you clarify what kind of operations are tracked?

In non-privileged containers, disk limit is determined by the amount of capacity that is used by the container fs.

Reply:
Hmm, I think I confused this with QOS limits. Let me do some digging about capacity limits.

Reply:
Added some notes and a TODO to revisit after some investigations.

Reply:
Can you also add any todos to the beta graduation crtieria so we don't forget?